map-style-change/cloudformation/template.yml

```diff
@@ -1,72 +1,72 @@
 AWSTemplateFormatVersion: "2010-09-09"
-Description: A Template to provision Amazon Location Service resources for map-style-change-example
+Description: A Template to provision Amazon Location Service resources for js-map-style-change
 Resources:
-  MapLight:
-    Type: AWS::Location::Map
-    Properties:
-      Configuration:
-        Style: VectorOpenDataStandardLight
-      Description: Map Resource created for map-style-change-example
-      MapName: map-style-change-example-1
-  MapDark:
-    Type: AWS::Location::Map
-    Properties:
-      Configuration:
-        Style: VectorOpenDataStandardDark
-      Description: Map Resource created for map-style-change-example
-      MapName: map-style-change-example-2
-  IdentityPool:
-    Type: AWS::Cognito::IdentityPool
-    Properties:
-      AllowUnauthenticatedIdentities: True
-      IdentityPoolName: sample-app-6
+      MapLight:
```

Comment:
I notice that there're extra spaces in all PRs has been raised so far. I will make it's harder for reviewer to focus on what has been really changed. Can you help commit new revision which only show the real change? I'm good to review as is given this indent setting will be used from now on (future CRs won't show changes for indent updates). Kindly let me know which option you have decided to go with.

Reply:
Hey @vinguthh , I'm following the same indent setting are being used in the marketing and demo repo's as well. Let keep it as is if that's alright with you.

Yes the same indentation will be used for all PRs.

Reply:
yes, that should work. As long as we stick with one indentation to avoid the change noice in future CRs.

Reply:
I check default indent settings on idea, some docs, and Amazon CFN yaml. They all suggest to use 2 as indent for YAML files. Please refer to this as one of the doc: https://docs.saltproject.io/en/3000/topics/troubleshooting/yaml_idiosyncrasies.html

Could you help use that as default indent for yaml files and update the PR? 🙏

Reply:
sure thing i'll look into it and update the PR

Reply:
@vinguthh updated